app/src/main/java/com/github/se/gomeet/MainActivity.kt

```diff
@@ -59,10 +62,16 @@ class MainActivity : ComponentActivity() {
               }
             }
             composable(Route.EXPLORE) { Explore(navAction) }
-            composable(Route.EVENTS) { Events(navAction) }
+            composable(Route.EVENTS) { Events(navAction, EventViewModel()) }
             composable(Route.TRENDS) { Trends(navAction) }
             composable(Route.CREATE) { Create(navAction) }
             composable(Route.PROFILE) { Profile(navAction) }
+            composable(Route.PRIVATE_CREATE) {
+              PrivateCreate(navAction, EventViewModel(userIdState.value))
+            }
+            composable(Route.PUBLIC_CREATE) {
+              PublicCreate(navAction, EventViewModel(userIdState.value))
+            }
```

Comment:
would suggest merging the two classes PrivateCreate and PublicCreate since they should be fairly similar